ios/brave-ios/Sources/Brave/Frontend/Sync/BraveCore/BraveSyncAPIExtensions.swift

```diff
@@ -117,6 +118,49 @@ extension BraveSyncAPI {
     }
   }
 
+  /// Stores a token that is NOT backed up when the user backs up to iCloud
+  /// This lets us track whether the app's data has been restored and sync chain should be reset
+  /// Returns true if the user should reset their sync chain, as they restored from iCloud backup
+  /// Returns false if they should not reset their sync chain
+  @MainActor
+  func setupDeviceRestorationTracking() async throws -> Bool {
```

Comment:
nit: basic API naming, named as if it performs a task that isn't supposed to return a value, but value returned makes it sound like its checking state. 

possible replacements: `isSyncChainValidForDevice` (boolean reversed), `isSyncChainInvalidForDevice`, `isFirstLaunchAfterDeviceRestore` (reversed), etc.

Reply:
Fixed. I chose `isSyncChainFromCloudRestoration` instead.